jest.config.js

```diff
@@ -8,6 +8,7 @@ const customJestConfig = {
   moduleDirectories: ['node_modules', '<rootDir>/'],
   moduleNameMapper: {
     '^@/components/(.*)$': '<rootDir>/src/components/$1',
+    '^@/layouts/(.*)$': '<rootDir>/src/layouts/$1',
```

Comment:
I'm curious, how can we use this?

Reply:
@hoangmirs because of this one 🥲

![Screen Shot 2565-06-08 at 13 38 56](https://user-images.githubusercontent.com/749672/172548937-2e74c937-99ee-4965-9431-cad8e13c252e.png)